index.js

```diff
@@ -147,6 +147,15 @@ module.exports = function(content) {
 
  	return exportsString + content.replace(/xxxHTMLLINKxxx[0-9\.]+xxx/g, function(match) {
 		if(!data[match]) return match;
+		
+		var urlToRequest;
+
+		if (config.interpolate === 'require') {
+			urlToRequest = data[match];
+		} else {
+			urlToRequest = loaderUtils.urlToRequest(data[match], root);
+		}
+		
 		return '" + require(' + JSON.stringify(loaderUtils.urlToRequest(data[match], root)) + ') + "';
```

Comment:
```diff
- return '" + require(' + JSON.stringify(loaderUtils.urlToRequest(data[match], root)) + ') + "';
+ return '" + require(' + JSON.stringify(urlToRequest) + "';
```

? Or I'm missing something, but it looks like  `urlToRequest` isn't used atm :)

Reply:
I was trying to keep existing functionality as sometimes `config.interpolate` could be `true` instead of `require`. I only wanted to change this for the instance I know it's broken.

Reply:
How is variable `urlToRequest` applied atm, from looking at the code it seems to always return atm

```js
'" + require(' + JSON.stringify(loaderUtils.urlToRequest(data[match], root)) + ') + "';
```

? So your changes don't apply ?

Reply:
You're correct; my mistake!